web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -166,10 +166,18 @@ public function getDefaultParamValue($type, $params) {
         break;
 
       case 'tags':
+        if (!isset($paramsDecoded['filters']['tags'])) {
+          $defaultParam = NULL;
+          break;
+        }
         $tid = (int) $paramsDecoded['filters']['tags'][0];
         $defaultParam = $this->entityTypeManager()->getStorage('taxonomy_term')->load($tid);
         break;
 
+      case 'limit':
+        $defaultParam = (empty($paramsDecoded['limit'])) ? 0 : $paramsDecoded['limit'];
```

Comment:
This works, but I also think you can case it to an int and all cases will be covered:
`$defaultParam = (int) $paramsDecoded['limit'];`

Reply:
Fixed.